methods/concat/index.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Appends the elements of `otherArrays` into the given array.
+ *
+ * @param {Array} ary
+ * @param {...*} otherArrays
+ *
+ * @return {Array} Returns the array itself.
+ *
+ * @example
+ * concat([1, 2, 3], [4, 5, 6]); // => [1, 2, 3, 4, 5, 6]
+ * concat(['a', 'b'], ['c'], ['d', 'e']) // => ['a', 'b', 'c', 'd', 'e']
+ *
+ * @example
+ * rbjs([1, 2]).concat([3, 4]); // => [1, 2, 3, 4]
+ * rbjs(['a', 'b']).concat(['c', 'd'], ['e'])  // => ['a', 'b', 'c', 'd', 'e']
+ */
+export default function concat(ary, ...otherArrays) {
+    otherArrays.forEach(otherArray => {
+        try {
+            ary.push(...otherArray);
+        } catch (e) {
+            if (e instanceof TypeError) {
+                throw new TypeError('Members of `otherArrays` must be an array instance');
```

Comment:
@aldnav Perhaps the item can be checked first, then throw an error if it's not the correct type, before passing it to `array.push`. Wrapping it in a `try-catch` block doesn't clearly communicate the type-checking intention for me.

Reply:
@arnellebalane Thank you for clearing that out. Updated here https://github.com/arnellebalane/rbjs/pull/40/commits/343df482a98264d218ae65435ae0066c0f8c3cef